front/src/views/PatientSignUpView/PatientSignUpView.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+
+import style from './patient_sign_up_view.scss';
+import { Button, TextInput } from '../../components';
+import Svg from 'svg-inline-react';
+import registrationIcon from '../../../assets/icons/user-tie.svg';
+
```

Comment:
Please don't write two component in one file.  If you want have this in one file I recommend you to use new render function such as renderInputWithLabel(args)
Although naming this component Input doesn't appeal to me. We have something similar to it. TextInput!